src/UI/ChestWindow.h

```diff
@@ -35,7 +35,7 @@ class cChestWindow :
 
 protected:
 	cWorld * m_World;
-	int m_BlockX, m_BlockY, m_BlockZ;  // Position of the chest, for the window-close packet
+	Vector3i m_Block;  // Position of the chest, for the window-close packet
```

Comment:
Rename this to avoid confusion about the contents (is it the blocktype, or the blockhandler?) `m_BlockPos` would be nice.